Framework/Kernel/inc/MantidKernel/UsageService.h

```diff
@@ -57,8 +66,12 @@ class MANTID_KERNEL_DLL UsageServiceImpl {
   void setInterval(const uint32_t seconds = 60);
   /// Registers the Startup of Mantid
   void registerStartup();
-  /// Registers the use of a feature in mantid
-  void registerFeatureUsage(const std::string &type, const std::string &name,
+  /// Registers the use of a feature in mantid. Just support a version using
```

Comment:
Yes good point about still being able to supply the old format. I'll add the overload. I deliberately hadn't added the overload to avoid any lingering instances of the old format but as you say we haven't closed that off